crates/voicevox_core_c_api/cbindgen.toml

```
@@ -34,4 +34,4 @@ rename_variants = "ScreamingSnakeCase"
 
 [parse]
 parse_deps = true
-include = ["voicevox_core"]
+include = ["sharevox_core"]
```

予想ですが、Rust APIのグレート名はそのままなのでそれで`SharevoxResultCode`を見失ってるんじゃないかと

```suggestion
include = ["voicevox_core"]
```

---

ありがとうございます...!
修正しました...!(Suggestionを取り入れられなくてごめんなさい...!)